src/etc/inc/dyndns.class

```diff
@@ -590,6 +602,87 @@
 					curl_setopt($ch, CURLOPT_URL, $server);
 					curl_setopt($ch, CURLOPT_POSTFIELDS, $post_data);
 					break;
+				case 'hetzner':
+				case 'hetzner-v6':
+					// API documentation: https://dns.hetzner.com/api-docs
+					$server = 'https://dns.hetzner.com/api/v1';
+					$fqdn = str_replace(' ', '', $this->_dnsHost);
+					$recordType = $this->_useIPv6 ? "AAAA" : "A";
+					$ttlData = intval($this->_dnsTTL) < 1 ? 120 : intval($this->_dnsTTL);
+					$hostData = [
+						"value" => "{$this->_dnsIP}",
+						"type" => $recordType,
+						"name" => "",
+						"ttl" => $ttlData,
+						"zone_id" => ""
+					];	
+
+					$headerAuth = [
+						"Auth-API-Token: {$this->_dnsPass}",
+						'Content-Type: application/json'
+					];
+		
+					curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+					curl_setopt($ch, CURLOPT_HTTPHEADER, $headerAuth);
+
+					// Get all zone info
+					$zonesUrl = "$server/zones";
```

Comment:
Variable references inside a string need to be surrounded by curly braces to eliminate ambiguity, so this should be:
```php
$zonesUrl = "{$server}/zones";
```

Reply:
done